modules/ztp-precaching-downloading-artifacts.adoc

```diff
@@ -38,19 +38,19 @@ To remove this restriction, you can precede your commands with `taskset 0xffffff
 [id="ztp-preparing-ocp-images_{context}"]
 == Preparing to download the {product-title} images
 
-To download {product-title} container images, you need to know the multicluster engine (MCE) version. When you use the `--du-profile` flag, you also need to specify the {rh-rhacm-first} version running in the hub cluster that is going to provision the {sno}.
+To download {product-title} container images, you need to know the {mce-short} version. When you use the `--du-profile` flag, you also need to specify the {rh-rhacm-first} version running in the hub cluster that is going to provision the {sno}.
```

Comment:
Again, is "multicluster engine operator version" the same as "multicluster engine version"?